source/_integrations/zwave_js.markdown

```diff
@@ -161,7 +162,8 @@ While your Z-Wave mesh is permanently stored on your dongle, the additional meta
 Do this before using the device with another controller, or when you don't use the device anymore. It removes the device from the Z-Wave network stored on the controller. It also removes the device and all its entities from Home Assistant. You can not join a device to a new network if it is still paired with a controller.
 
 1. In Home Assistant, go to {% my integrations title="**Settings** > **Devices & services**" %}.
-2. Select the **Z-Wave** integration. Then, select **Configure**.
+2. Select the **Z-Wave** integration.
+   - Then, on the entry of the hub, select {% icon "ic:baseline-arrow-forward-ios" %} to open the device info page.
```

Comment:
I'm checking this with 2025.7.0b3, and in that version I think this should be "select the cogwheel", since the "Remove a device" button now only exists in the config panel.